packages/grafana-ui/src/components/Icon/Icon.tsx

```diff
@@ -14,6 +14,7 @@ export interface IconProps extends React.HTMLAttributes<HTMLDivElement> {
   size?: IconSize;
   type?: IconType;
   title?: string;
+  ariaLabel?: string;
```

Comment:
this prop is not required because it should already included in `React.HTMLAttributes<HTMLDivElement>`, and spread onto the div via the rest `...divElementProps`

Reply:
Ah, I see, I wasn't aware of that. I've pushed a new commit. Thank you so much for your feedback.